resource/geo-density-chart.js

```diff
@@ -84,23 +84,31 @@ function GeoDensityChart(){
         // iterate in the layers of the map
         _mapPointer.eachLayer(function(layer){
             if (layer.options.id){
+                console.log("[MapTile] Received previous maptile id:", layer.options.id);
                 let url;
-                _mapPointer.removeLayer(layer);
+                let idTileMap;
```

Comment:
You could perhaps rename `idTileMap` to sth like `newMapBackground` because for me, this name makes it difficult to understand that it's actually the map background colour.

Reply:
I renamed `idTileMap` to `newBaseMap`, I think the light_all is more than just the color of the map it can be an id for an exotic map background

Reply:
Done in d2c7118